carta/image.py

```diff
@@ -593,6 +593,181 @@ def hide_contours(self):
+
+        This automatically enables use of the vector overlay colormap.
+
+        Parameters
+        ----------
+        colormap : {0}
+            The colormap.
+        bias : {1}
+            The colormap bias.
+        contrast : {2}
+            The colormap contrast.
+        """
+        self.call_action("vectorOverlayConfig.setColormap", colormap)
+        self.call_action("vectorOverlayConfig.setColormapEnabled", True)
+        if bias is not None:
+            self.call_action("vectorOverlayConfig.setColormapBias", bias)
+        if contrast is not None:
+            self.call_action("vectorOverlayConfig.setColormapContrast", contrast)
+
+    def apply_vector_overlay(self):
+        """Apply the vector overlay configuration."""
+        self.call_action("applyVectorOverlay")
+
+    @validate(NoneOr(Constant(VectorOverlaySource)), NoneOr(Constant(VectorOverlaySource)), NoneOr(Number()), Boolean(), NoneOr(Number()), NoneOr(Number()), NoneOr(Number()), Number(), NoneOr(Number()), NoneOr(Number()), Number(), Number(), Number(), NoneOr(Color()), NoneOr(Constant(Colormap)), NoneOr(Number()), NoneOr(Number()))
+    def plot_vector_overlay(self, angular_source=None, intensity_source=None, pixel_averaging=4, fractional_intensity=False, threshold=None, q_error=None, u_error=None, thickness=1, intensity_min=None, intensity_max=None, length_min=0, length_max=20, rotation_offset=0, color=None, colormap=None, bias=None, contrast=None):
+        """Set the vector overlay configuration, styling and colour or colourmap; and apply vector overlay; in a single step.
+
+        If both a colour and a colourmap are provided, the colourmap will be visible.
+
+        Parameters
+        ----------
+        angular_source : {0}
+            The angular source. If the image is with Stoke information, ``Computed PA`` is set by default; If the image is without Stoke information, ``Current image`` is set.
+        intensity_source : {1}
+            The intensity source. If the image is with Stoke information, ``Computed PI`` is set by default; If the image is without Stoke information, ``Current image`` is set.
+        pixel_averaging : {2}
+            The pixel averaging width in pixel. Set to ``None`` to disable pixel averaging.
+        fractional_intensity : {3}
+            Enable fractional polarization intensity. By default the absolute polarization intensity is used.
+        threshold : {4}
+            The threshold in Jy/pixels. By default is 4.
+        q_error : {5}
+            The Stokes Q error in Jy/beam. Set both this and ``u_error`` to enable debiasing. Debiasing is disabled by default.
+        u_error : {6}
+            The Stokes U error in Jy/beam. Set both this and ``q_error`` to enable debiasing. Debiasing is disabled by default.
+            thickness : {0}
+            The line thickness in pixels. By default is 1.
+        intensity_min : {1}
+            The minimum value of intensity in Jy/pixel.
+        intensity_min : {2}
```

Comment:
This numbering was also a bug in my contour PR -- I just fixed it. You need to renumber these placeholders to match the order of the parameters in this method.

`thickness` also needs to be de-indented by one level.